### PR TITLE
Part of #16033: VehiclePaint constants

### PR DESCRIPTION
**Pull request by tfai8308**
I used my best judgment on what each sprite should be named based on what the actual sprites looked like in g1.dat and added in all the relevant water sprites from 29014-29109. Also a water particle spray from 22965-22972.

As far as I can tell though, some of the ids (e.g. 29014, 29046) are used in a few places, while some (e.g. 29022, 29054, etc.) aren't actually used explicitly. I'm not sure if this is an issue to add the non-explicitly-used ids into sprites.h, but figured they couldn't hurt to have around.

### CODE REVIEW

**Comment by tfai8308** [comment]
Hi, it's been a little while, but can someone let me know if I am on the right track with this? I would definitely appreciate any feedback that anyone has about this.

File: src/openrct2/sprites.h [hunk 1]
```diff
@@ -754,6 +763,114 @@ enum
...
     SPR_NEW_RIDE_MASK = 29013,
 
+    SPR_SPLASH_EFFECT_1_NE_0 = 29014,
+    SPR_SPLASH_EFFECT_1_NE_1 = 29015,
+    SPR_SPLASH_EFFECT_1_NE_2 = 29016,
+    SPR_SPLASH_EFFECT_1_NE_3 = 29017,
+    SPR_SPLASH_EFFECT_1_NE_4 = 29018,
+    SPR_SPLASH_EFFECT_1_NE_5 = 29019,
+    SPR_SPLASH_EFFECT_1_NE_6 = 29020,
+    SPR_SPLASH_EFFECT_1_NE_7 = 29021,
```
Comment (Broxzier) [inline src/openrct2/sprites.h:773]:
```suggestion
    SPR_SPLASH_EFFECT_1_NE_0 = 29014,
    SPR_SPLASH_EFFECT_1_NE_1 = SPR_SPLASH_EFFECT_1_NE_0 + 1,
    SPR_SPLASH_EFFECT_1_NE_2 = SPR_SPLASH_EFFECT_1_NE_0 + 2,
    SPR_SPLASH_EFFECT_1_NE_3 = SPR_SPLASH_EFFECT_1_NE_0 + 3,
    SPR_SPLASH_EFFECT_1_NE_4 = SPR_SPLASH_EFFECT_1_NE_0 + 4,
    SPR_SPLASH_EFFECT_1_NE_5 = SPR_SPLASH_EFFECT_1_NE_0 + 5,
    SPR_SPLASH_EFFECT_1_NE_6 = SPR_SPLASH_EFFECT_1_NE_0 + 6,
    SPR_SPLASH_EFFECT_1_NE_7 = SPR_SPLASH_EFFECT_1_NE_0 + 7,
```

**Comment by Broxzier** [comment]
The changes so far look alright. One suggestion is to use relative IDs starting after the first, like how it's done in similar cases.

https://github.com/OpenRCT2/OpenRCT2/blob/34c7bed/src/openrct2/sprites.h#L288-L291

**Comment by tfai8308** [comment]
Thanks for the response. I've changed it to use relative IDs over absolute ones to line up better with the rest of the file.

You're correct too, this was supposed to be a draft pr pending I received feedback from somebody. But since I was recommended to make prs for single files at a time, the constants in VehiclePaint.cpp should be taken care of, and you've provided some feedback already, I think these changes can be merged if there's nothing else for these files specifically.

**Review by Gymnasiast — APPROVED** [review]

**Review by Broxzier — APPROVED** [review]

**Pull request merged**